ChatSecure/Classes/Model/Yap Storage/OTRYapDatabaseObject.m

```diff
@@ -29,7 +29,7 @@ - (instancetype)initWithUniqueId:(NSString *)uniqueId
 - (void)saveWithTransaction:(YapDatabaseReadWriteTransaction *)transaction
 {
     NSString *collection = self.class.collection;
-    [transaction setObject:self forKey:self.uniqueId inCollection:collection];
+    [transaction setObject:self forKey:self.uniqueId inCollection:collection withMetadata:[transaction metadataForKey:self.uniqueId inCollection:collection]];
```

Comment:
Where is metadata being used?

Reply:
Not needed now, no, but if we DO start using meta upstream then we will want this, otherwise it will be discarded.